agent/api/ecsclient/client_test.go

```diff
@@ -499,7 +505,7 @@ func TestRegisterContainerInstanceNoIID(t *testing.T) {
 			assert.True(t, ok, `Could not find resource "PORTS_UDP"`)
 			assert.Equal(t, "STRINGSET", *resource.Type, `Wrong type for resource "PORTS_UDP"`)
 			// 3 from expectedAttributes and 2 from additionalAttributes
```

Comment:
Can you update this comment before you merge the pr?

Reply:
Yes.